### ISSUE TITLE
Vehicle attachments bug

### ISSUE THREAD

**Issue by Alfiethesquire** [citizenparker/swsheets#252]
I’ve been using swsheets.com for a long time, since about 2016-17, and Never really made a vehicle statblock on the site, until today when I tried making a modded out ship for a campaign and found I could only have one attachment in the attachment section? I don’t know if anyone else has brought up this bug but it’s a pretty big part of the ships stats if players decide to mod it.

Hope this bug gets resolved, cheers for taking the time to read through my short little paragraph.

**Comment by duffn** [comment]
Hi @Alfiethesquire, thanks for the note.

Perhaps I'm misunderstanding, but you can add more than one attachment in the attachments section with the `Add Attachment` button.

![Screen Shot 2020-03-20 at 12 26 47 PM](https://user-images.githubusercontent.com/3457341/77194753-210f7d00-6aa6-11ea-833a-3b5882a012a5.png)

![Screen Shot 2020-03-20 at 12 26 55 PM](https://user-images.githubusercontent.com/3457341/77194762-253b9a80-6aa6-11ea-9ee9-5970f0550237.png)

Let me know if this is not what you're talking about.

**Comment by Alfiethesquire** [comment]
Ahh yeah it’s just me then, but for some reason I keep pressing it and it ain’t adding

**Comment by duffn** [comment]
Well, if it's not working for you, that's a definite bug then and maybe we can fix it.

What browser are you using? This happens every time? Does it happen after refreshing the page?

**Comment by duffn** [comment]
I'm closing this due to inactivity, but let me know if this is still happening to you and if you can provide more information. Thanks for using the site!

**Comment by Alfiethesquire** [comment]
Ahh it’s fine mate, don’t think I ever resolved the issue but I think I was
an isolated incident. Turns out the game I was prepping for didn’t go ahead.

But thank you anyways for looking into, I do love swsheets, its the only
character sheet site I use, it has always worked perfectly for me except
with one bug I reported where the delete button didn’t actually delete the
character, but even then the help has always been great. 🙏🏻
